pkg/process/checks/net.go

```diff
@@ -184,6 +184,56 @@ func getConnectionsByPID(conns *model.Connections) map[int32][]*model.Connection
 	return result
 }
 
+func convertDNSEntry(dnstable map[string]*model.DNSDatabaseEntry, namemap map[string]int32, namedb *[]string, ip string, entry *model.DNSEntry) {
+	dbentry := &model.DNSDatabaseEntry{}
```

Comment:
Can we set the capacity of `NameIndexes` here since we know it will be the same as `entry.Names`?